protocol/src/main.ts

```diff
@@ -23,7 +23,7 @@ export {
 	Message, NotificationMessage, RequestMessage, RPCMessageType,
 	RequestType, RequestType0, RequestHandler, RequestHandler0, GenericRequestHandler, StarRequestHandler, HandlerResult,
 	NotificationType, NotificationType0, NotificationHandler, NotificationHandler0, GenericNotificationHandler, StarNotificationHandler,
-	MessageReader, PartialMessageInfo, MessageWriter, Logger, ConnectionStrategy,
+	MessageReader, PartialMessageInfo, MessageWriter, Logger, ConnectionStrategy, ConnectionOptions,
```

Comment:
We should also export the strategies here so that users can still only prereq the protocol with needing to prereq json-rpc